CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.5.0 (2017-11-03)
+
+- Added `SkyAppOmnibarProvider` to enable custom `envId` and `svcId` to be provided to the Omnibar. [#323](https://github.com/blackbaud/skyux-builder/pull/323)
+- Cleaned up logging by only generating config for known commands. [#321](https://github.com/blackbaud/skyux-builder/pull/321)
+- Enabling default `envid` and `svcid` params via `skyuxconfig.json`. [#320](https://github.com/blackbaud/skyux-builder/pull/320)
```

Comment:
I'd change "Enabling" to "Enabled" to consistently use past tense.

I'd also change "via" to either "through" or "in." (At some point in the past, I was told to avoid Latin words such as "via" to make localization/translation easier. Not sure if that's still something to worry about, but I tend to avoid "via" and similar Latin phrases just to be safe.)